### PR TITLE
MGMT-1603 Prevent dummy iso generation uploading to s3

### PR DESCRIPTION
**Pull request by filanov**
We only need to download the docker image to the cluster so there is no reason
to let the dummy job upload anything to s3.
Overwrite the default command into 'echo pass'

### CODE REVIEW

**Comment by filanov** [comment]
```
kubectl logs dummyimage-00000000-0000-0000-0000-000000000000-2020072216pt8zw
pass
```

File: internal/bminventory/inventory.go [hunk 1]
```diff
@@ -193,7 +194,12 @@ func getQuantity(s string) resource.Quantity {
 }
 
 // create discovery image generation job, return job name and error
-func (b *bareMetalInventory) createImageJob(jobName, imgName, ignitionConfig string) *batch.Job {
+// cmd is optional to overwrite default command
+func (b *bareMetalInventory) createImageJob(jobName, imgName, ignitionConfig string, cmd *string) *batch.Job {
+	var command []string
+	if cmd != nil {
+		command = strings.Split(swag.StringValue(cmd), " ")
+	}
```
Comment (avishayt) [inline internal/bminventory/inventory.go:202]:
```suggestion
func (b *bareMetalInventory) createImageJob(jobName, imgName, ignitionConfig string, upload boolean) *batch.Job {
	var command []string
	if !upload {
		command = strings.Split(swag.StringValue("echo pass"), " ")
	}
```

Comment (avishayt) [reply internal/bminventory/inventory.go:202]: s/upload/performUpload/

**Comment by filanov** [comment]
> Would something like that be more readable?

Yes good option, thanks

File: internal/bminventory/inventory.go (hunk 1 above)
Comment (filanov) [reply internal/bminventory/inventory.go]: fixed.

**Pull request merged**